runtime/src/precompiles.rs

```diff
@@ -2,13 +2,58 @@ use pallet_evm::{Precompile, PrecompileHandle, PrecompileResult, PrecompileSet};
 use sp_core::H160;
 use sp_std::marker::PhantomData;
 
+use pallet_evm_precompile_blake2::Blake2F;
+use pallet_evm_precompile_bn128::{Bn128Add, Bn128Mul, Bn128Pairing};
+use pallet_evm_precompile_ed25519::Ed25519Verify;
 use pallet_evm_precompile_modexp::Modexp;
 use pallet_evm_precompile_sha3fips::Sha3FIPS256;
 use pallet_evm_precompile_simple::{ECRecover, ECRecoverPublicKey, Identity, Ripemd160, Sha256};
+use pallet_evm_precompile_sr25519::Sr25519Precompile;
+use pallet_evm_precompile_substrate_ecdsa::SubstrateEcdsaPrecompile;
 use pallet_evm_precompile_xvm::XvmPrecompile;
 
 pub struct FrontierPrecompiles<R>(PhantomData<R>);
 
+pub mod consts {
+	use sp_core::H160;
+
+	pub const EC_RECOVER: H160 = hash(1);
+	pub const SHA256: H160 = hash(2);
+	pub const RIPEMD160: H160 = hash(3);
+	pub const IDENTITY: H160 = hash(4);
+	pub const MODEXP: H160 = hash(5);
+	pub const BN128_ADD: H160 = hash(6);
+	pub const BN128_MUL: H160 = hash(7);
+	pub const BN128_PAIRING: H160 = hash(8);
+	pub const BLAKE2F: H160 = hash(9);
+	pub const SHA3_FIPS256: H160 = hash(0x400);
+	/// 402 is used in Astar and Moonbeam, so preserve the address for contracts interoperability
+	pub const EC_RECOVER_PUBLIC_KEY: H160 = hash(0x402);
+	/// 403 is also used in Astar, so preserve the address for contracts interoperability.
+	/// TO DECIDE: Celo uses 0xF3 for it, so we need to decide which one to use.
```

Comment:
Let's go with Celo's address... this will make it easier to use the Celo IBC smart contracts out-of-the-box.